tests/test_actions.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the `edx-enterprise` admin actions module.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import unittest
+
+import factory
+import mock
+
+from enterprise.actions import export_as_csv_action
+
+
+class DummyModel(object):
```

Comment:
@clintonb It [is used](https://github.com/edx/edx-enterprise/pull/1/files#diff-cb0d755b9f3ef2c188945841d6605f1dR38) with `EnterpriseCustomer` - I just don't want to use `EnterpriseCustomer` for testing it for the reasons I've mentioned above